src/ESports.ts

```diff
@@ -0,0 +1,152 @@
+import fetch from "node-fetch";
+import Discord = require("discord.js");
+import { SharedSettings } from "./SharedSettings";
+import * as CheerioAPI from "cheerio";
+import * as momentjs from "moment";
+import Botty from "./Botty";
+
+interface ESportsAPIReturnData {
+    resultsHtml: string;
+    fixturesHtml: string;
+    resultsMonths: string;
+    fixturesMonths: string;
+}
+
+interface ESportsLeagueSchedule {
+    league: string;
+    time: string;
+    teamA: string;
+    teamB: string;
+}
+
+export default class ESportsAPI {
+    private bot: Discord.Client;
+    private settings: SharedSettings;
+
+    // Map<String, Map<String, ESportsLeagueSchedule>>
+    private schedule: { [date: string]: { [league: string]: ESportsLeagueSchedule[] } } = {};
```

Comment:
You could always use real maps here ;) Javascript has em.

Reply:
True, its just a remnant of not being able to use maps with the FileBackedObject